src/js/content_scripts/input-mapper-background-utils.js

```diff
@@ -49,4 +49,30 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             chrome.runtime.sendMessage(actionData);
         }
     };
+
+    /**
+     *
+     * Sends message to the background script to zoom in/out on the webpage using
+     * thumbsticks.
+     *
+     * @param {Object} that - The inputMapper component.
+     * @param {Integer} value - The value of the gamepad input.
+     * @param {Boolean} invert - Whether the zooming should be in opposite order.
+     *
+     */
+    gamepad.inputMapperUtils.background.thumbstickZoom = function (that, value, invert) {
+        // Get the updated input value according to the configuration.
+        var inversionFactor = invert ? -1 : 1;
+        value = value * inversionFactor;
+        var zoomType = value > 0 ? "zoomOut" : "zoomIn",
```

Comment:
I really suspect you're going to want some kind of cutoff for this, like 0.25 in either direction.  Many PS4 controllers especially have jitter in about that range, which would be really disconcerting in this case.  For now I'd say add another TODO to discuss when we work on #22.

Reply:
I've added the cutoff check just below it. You can check that [here](https://github.com/fluid-lab/gamepad-navigator/pull/42/files/5022f4bccfa2fb5bd4cad25e2fc6210e166a57eb#diff-fee08b1acaa59bc549d236d3f4c9a80bR74). So the jittery stuff would be handled in any case.

Reply:
Ah!  Thanks.